src/cluster_slot_stats.c

```diff
@@ -43,6 +43,8 @@ static uint64_t getSlotStat(int slot, int stat_type) {
     uint64_t slot_stat = 0;
     if (stat_type == KEY_COUNT) {
         slot_stat = countKeysInSlot(slot);
+    } else if (stat_type == NETWORK_BYTES_IN) {
+        slot_stat = server.cluster->slot_stats[slot].network_bytes_in;
```

Comment:
```suggestion
    } else if (stat_type == NETWORK_BYTES_IN) {
        slot_stat = server.cluster->slot_stats[slot].network_bytes_in;
    } else if (stat_type == NETWORK_BYTES_OUT) {
        slot_stat = server.cluster->slot_stats[slot].network_bytes_out;
```
We might do a case switch here so that it throws a warning if we miss a type here.